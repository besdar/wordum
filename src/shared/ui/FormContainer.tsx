import React from 'react';
import {Grid} from './Grid';
import {ScrollView} from 'react-native';

type Props = {
  children: React.ReactNode;
};

export const FormContainer = ({children}: Props) => (
  <ScrollView>
    <Grid direction="column" rowGap={10} padding={5} alignItems="stretch">
      {children}
    </Grid>
  </ScrollView>
);
