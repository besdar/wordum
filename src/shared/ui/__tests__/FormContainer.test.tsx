import {render} from '@testing-library/react-native';
import {FormContainer} from '../FormContainer';
import React from 'react';

describe('FormContainer', () => {
  it('should match the snapshot', () => {
    const {toJSON} = render(<FormContainer>Test</FormContainer>);

    expect(toJSON()).toMatchSnapshot();
  });
});
