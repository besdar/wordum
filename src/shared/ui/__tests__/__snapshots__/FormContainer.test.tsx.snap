// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormContainer should match the snapshot 1`] = `
<RCTScrollView>
  <View>
    <View
      style={
        [
          {
            "display": "flex",
            "width": "100%",
          },
          {
            "alignItems": "stretch",
            "columnGap": undefined,
            "flexBasis": undefined,
            "flexDirection": "column",
            "flexWrap": "nowrap",
            "gap": undefined,
            "justifyContent": "flex-start",
            "padding": 5,
            "rowGap": 10,
          },
        ]
      }
    >
      Test
    </View>
  </View>
</RCTScrollView>
`;
